README.md

```diff
@@ -0,0 +1,50 @@
+# HAMi Community
+
+Welcome to the HAMi Community!
+
+This is the starting point for joining and contributing to the HAMi community—improving documentation, code, giving talks, etc.
+
+## Community Meeting
+
+Regular Community Meetings:
+- **Asia Time**: **Friday at 16:00 (UTC+8)** (weekly).
+  ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=14%3A30&tz=GMT%2B8&))
+- **Europe Time**: **Thursday at 11:00 (UTC+1)** (biweekly).
```

Comment:
We haven't Europe Community Meeting

Reply:
I submitted a PR and made some optimizations to the readme

https://github.com/Project-HAMi/community/pull/7/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
I also considered updating the CONTRIBUTOR-LADDER.md, but felt that there was a difference between ladder and contributing, so I kept them separate.